Formula/yaegi.rb

```diff
@@ -0,0 +1,26 @@
+class Yaegi < Formula
+  desc "Yet another elegant Go interpreter"
+  homepage "https://github.com/containous/yaegi"
+  url "https://github.com/containous/yaegi/archive/v0.7.2.tar.gz"
+  sha256 "1f2e63290614e26ffb82ef16af23d0ce5237c10a19a5c1191c22398adb62bc1e"
+  head "https://github.com/containous/yaegi.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GO111MODULE"] = "on"
```

Comment:
Is it really necessary since we use go 1.13.5 for building formulae and `GO111MODULE` defaulted to `auto` for it?

Reply:
It is not required anymore. I removed it.